docs: update project finance doc with info about what we spend money on

derberg opened the pull request:
goal of this PR is to clarify in one MD file not only how people can sponsor us but also what we spend money on.

it will make it super easy for me to share it with different companies to have a good argument for them to be our sponsors

cc @thulieblack 

@alequetzalli 🙏🏼

derberg commented:
Initial text totally rewritten with https://quillbot.com/

I stopped paying for Grammarly. Before I go that path back again, I want to experiment with some other free offerings. that do more than spell check. Would be nice to have long term a list of Grammarly-alternatives that we can propose to docs-based contributions

thulieblack commented on PROJECT_FINANCE.md:
```diff
@@ -1,22 +1,75 @@
+
+### Hiring
+
+Our huge and expanding community necessitates a full-time dedication. We are able to sign contracts with people to work on the AsyncAPI Initiative thanks to the assistance of Open Collective. This year, [Thulie] (https://www.linkedin.com/in/v-thulisile-sibanda/) was hired as the initiative's community manager. In the long run, we would like to expand our staff.
```
```suggestion
Our huge and expanding community necessitates a full-time dedication. We are able to sign contracts with people to work on the AsyncAPI Initiative thanks to the assistance of Open Collective. This year, [Thulie](https://www.linkedin.com/in/v-thulisile-sibanda/) was hired as the initiative's community manager. In the long run, we would like to expand our staff.
```

thulieblack commented:
Honestly speaking, I'm not really convinced about using https://quillbot.com/. But then since it's just an experiment lets see ......

derberg commented:
@thulieblack curious why, gimme more 😄 the experiment to have meaning, should include feedback 😛 you did not like the text of changes I'm pushing here?

alequetzalli commented:
Ok @derberg I wrote a new draft and made that it's own commit: https://github.com/asyncapi/community/pull/673/commits/fdff0c61d6ff3317684643358e052ce804e00fe1

That said, this page would be improved if we added a few more items, which I outline below...

---


### Testimonials
Showcase the impact of the AsyncAPI Initiative by sharing testimonials from our community members, project beneficiaries, and industry leaders who have witnessed the initiative's positive influence on their projects or organizations.

### Sponsorship Tiers and Benefits
Introduce sponsorship tiers with corresponding benefits to attract potential sponsors. For example:
- Bronze: tier level includes... TBD
- Silver: tier level includes... TBD
- Gold: tier level includes... TBD

### Custom Sponsorship Packages
Do we offer custom sponsorship packages tailored to specific needs, interests, or desired visibility levels? If so, we should explain how this can attract sponsors who want a more personalized approach.

### Success Stories via Case Studies
Highlighting past achievements and milestones via our recently added case studies section demonstrates tangible outcomes which can convince new sponsors to contribute. We can also link to our new website case studies page.


### Call-to-Action
End the page with a compelling call-to-action, encouraging potential sponsors to take the next step and contribute to your cause. Right now, reaching the end of this page only shows a user text. We should include an actual button + copy that leads to action.

### Contact Information
Provide clear contact information, making it easy for potential sponsors to reach out with questions, suggestions, or partnership inquiries about sponsoring AsyncAPI. We should have an email at a minimum. 

---

## Improving Financial Transparency and User Experience
To ensure financial transparency and provide a seamless user experience for potential sponsors, we should offer a detailed breakdown of past and anticipated expenses, demonstrating how funds have been utilized and allocated. **While our financial records are accessible through Open Collective, manually scrolling through past expenses is cumbersome.** Instead, we can curate a list of representative expenses and include a summary table on this page for the community's convenience.

Read the below example of how we can apply this idea...

### Examples of Past Expenses
1. Hiring a community manager to coordinate community efforts and foster collaboration.
2. Hosting webinars and conferences, covering venue rentals, catering, and marketing costs.
3. Paying for essential tools and services, such as Zoom and Descript, to support daily operations.
4. Funding the Mentorship and Bounty programs, rewarding and supporting community members in their contributions.
5. Allocating resources for the development and maintenance of the Swag Store.

### Summary Table of Expenses

| Category                | Description                                       | Amount Spent (USD) | Percentage of Total Budget |
|-------------------------|---------------------------------------------------|--------------------|----------------------------|
| Personnel               | Salaries, contracts, and stipends                | 40,000             | 40%                         |
| Events                  | Webinars, conferences, and meetups               | 20,000             | 20%                         |
| Tools and Services      | Subscriptions, licenses, and hosting             | 15,000             | 15%                         |
| Programs                | Mentorship, Bounty, and Swag Store               | 15,000             | 15%                         |
| Marketing and Outreach  | Advertising, promotional materials, and social media campaigns | 10,000  | 10%                         |
| **Total**               |                                                   | **100,000**        | **100%**                    |

This improved presentation of financial information allows potential sponsors to quickly grasp our expense allocation, instilling trust and confidence in our financial management.

alequetzalli reviewed (changes requested):
Improve further based on provided feedback: 
https://github.com/asyncapi/community/pull/673#issuecomment-1515621813

derberg commented:
@alequetzalli Thanks so much for incredible review ❤️ 

I made some changes as per your suggestion. Also fyi, there will be also a much nicer promotional poster that we will use to advertise project sponsorship -> https://github.com/asyncapi/community/issues/672

> Testimonials

This will be super hard to accomplish. We do not have that many sponsors, and the ones I know sponsor for 2 reasons:
- they just want to give back and show their support
- they share info about AsyncAPI support in sales pich to customers

Both do not bring any direct benefit that is easy to spot and share as testimony.
But I definitely will not leave it, and long term have it on my agenda when I talk to sponsors. Interesting will be to see what thay have to say. Thanks!!!

> Sponsorship Tiers and Benefits

Added. I also created separate issue as current sponsorship tier is pretty old. We might want to increase the required amounts. I need to talk to each existing sponsor and check. We for example do social media posts but there is nothing in official benefits list

https://github.com/asyncapi/community/issues/684

> Custom Sponsorship Packages

We don't do anything special but could, will investigate when I will work on tiers refreshment as it will also require talking to sponsors

> Success Stories via Case Studies

I think you mean mention success stores thanks to sponsorship. I added it. Not case study page as it was not sponsorship money driven, but we have many other cool successes.

> Call-to-Action

Added button in `Sponsorship Tiers` section. 

> Contact Information

Also added contact section

> Improving Financial Transparency and User Experience

Definitely makes a lot of sense. Just at the moment there is no easy way I can do it. The thing is that we never did a proper budget planning, that is transparent and we can forsee expenses. This is something I plan for 2024. 

So I opened an issue to followup on that topic once I have all the numbers for this year, as for example I do not yet have final costs for swag store, and we are not yet 100% sure what will be final budget for Mentorship and Bounty. 

I do not want to rush and add just partial data in this PR.
https://github.com/asyncapi/community/issues/683

---

How does that sound?

Thanks again!!! And btw how did you like the language style that I got from the bot? I personally did not like it, I prefer my own but yeah, I'm not a native speaker, so what do I know 😆

derberg commented:
@alequetzalli you probably want to review the new part that I added. Do you agree that some parts of your feedback I mitigate as followup issues?

alequetzalli commented:
> Thanks again!!! And btw how did you like the language style that I got from the bot? I personally did not like it, I prefer my own but yeah, I'm not a native speaker, so what do I know 😆

I tested this tool out to be able to get back to you on my opinion. hehe 🙃 Honestly, I found it lacking. It writes a little stiffly, less fluid than what Grammarly can help you produce. I wouldn't recommend this one, I still find myself favoring to Grammarly or Hemmingway. 😄

alequetzalli reviewed (approved):
APPROVED 🚢 🚢 🚢 🚢 

**note**: The CTA button in the Conference section was only 100px width; it looked ridiculous and wasn’t legible. I edited the button to match the same width of the CTA button in the Sponsorship tiers; consistency matters. 😄
